lib/caasp.pm

```diff
@@ -156,12 +156,30 @@ sub check_reboot_changes {
 
 # Install a pkg in Kubic
 sub trup_install {
+    my $input = shift;
+    my ($unnecessary, $necessary);
+
     # rebootmgr has to be turned off as prerequisity for this to work
     script_run "rebootmgrctl set-strategy off";
-    my $package = shift;
-    trup_call("pkg install $package");
-    process_reboot 1;
-    assert_script_run("rpm -qi $package");
+
+    my @pkg = split(' ', $input);
+    foreach (@pkg) {
+        if (!script_run("rpm -q $_")) {
```

Comment:
Ok, minor optional. It's easier to read conditions with negations, so we could flip it:
```
if(script_run("...")) {
 $ necessary .= "$_ ";
} else {
  $unnecessary  .= "$_ ";
}
This also doesn't require new verification run and code look good already, so it's up to you.